static/app/views/performance/http/definitions.tsx

```diff
@@ -0,0 +1,68 @@
+// https://httpwg.org/specs/rfc9110.html#overview.of.status.codes
+import {t} from 'sentry/locale';
+
+export const HTTP_RESPONSE_STATUS_CODES = {
+  '100': t('Continue'),
+  '101': t('Switching Protocols'),
+  '102': t('Processing'),
+  '103': t('Early Hints'),
+  '200': t('OK'),
+  '201': t('Created'),
+  '202': t('Accepted'),
+  '203': t('Non-Authoritative Information'),
+  '204': t('No Content'),
+  '205': t('Reset Content'),
+  '206': t('Partial Content'),
+  '207': t('Multi-Status'),
+  '208': t('Already Reported'),
+  '226': t('IM Used'),
+  '300': t('Multiple Choices'),
+  '301': t('Moved Permanently'),
+  '302': t('Found'),
+  '303': t('See Other'),
+  '304': t('Not Modified'),
+  '305': t('Use Proxy'),
+  '306': t('unused'),
```

Comment:
🤔 is this intentionally lower cased?
```suggestion
  '306': t('Unused'),
```

Reply:
Oh, well spotted, I'm gonna remove this, it's literally _not in use_

Reply:
😆